recipes-extended/sudo/sudo_%.bbappend

```diff
@@ -0,0 +1,8 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+FILES_${PN} += "${sysconfdir}/sudoers"
+
+
+do_install_append() {
+    echo "user    ALL=(ALL:ALL) NOPASSWD:/bin/mv /tmp/mender.conf.tmp /etc/mender/mender.conf" >> ${D}${sysconfdir}/sudoers
+}
```

Comment:
How secure is this? Do we want to add a warning similar to what we have in the meta-mender-demo layer that this is not appropriate for production?

Reply:
I suppose we could add a `bbwarn "Insecure blah, blah"` like in meta-mender-demo. Not that I think anyone would run this in production though. :-)